app/app.js

```diff
@@ -36,7 +36,7 @@ app.set('view engine', 'njk')
 
 // Set up middleware to serve static assets
 app.use('/public', express.static(configPaths.public))
-app.use('/icons', express.static(path.join(configPaths.public, 'icons')))
+app.use('/@govuk-frontend/icons', express.static(path.join(configPaths.components, '/icons/')))
```

Comment:
Similar to the config paths, if we keep the paths generic 'component', others can use this code for their own review application easier.

Reply:
here i'm linking to the icons package directly. So there is no need to copy icons to the public folder as we do now

Reply:
funny that we're actually using only one image from there.. the one for the start button, aren't we?

Reply:
yes

Reply:
Sorry, this is a bad example to reply to, I meant more widely the paths.

Reply:
oh that. i'll have a look at express url adaption